src/Campaigns/resources/admin/components/CampaignsListTable/index.tsx

```diff
@@ -8,11 +8,24 @@ import {CampaignsRowActions} from './CampaignsRowActions';
 import styles from './CampaignsListTable.module.scss';
 import {GiveCampaignsListTable} from './types';
 import CreateCampaignModal from '../CreateCampaignModal';
+import {useState} from "react";
 
 declare const window: {
     GiveCampaignsListTable: GiveCampaignsListTable;
 } & Window;
 
+/**
+ * Auto open modal if the URL has the query parameter id as new
+ *
+ * @unreleased
+ */
+const autoOpenModal = () => {
```

Comment:
@glaubersilva where are we using the auto-open feature? I see that we were triggering the modal this way using the "create campaign" button in the table's blank state, but I'm not sure where else it is used.

Reply:
@kjohnson It is being used only in the place you mentioned as far as I know.

Reply:
In that case, we might just need to remove the auto-open feature, unless we are deep-linking from somewhere else to create a campaign from a URL/hyperlink.

Reply:
@kjohnson I can imagine situations (like admin notices) where we want to redirect users directly to the modal to create a new campaign, so I would prefer to keep this feature in the codebase just in case.